Station/pycpt-seasonal-station-grid-Chile.ipynb

```diff
@@ -0,0 +1,794 @@
+    "    #   This field is defined by a python string, with two three-letter month name abbreviations \n",
+    "    #   whose first letters are capitalized, and all other letters are lowercase\n",
+    "    #   and who are separated by a dash character. \n",
+    "    #   For example, if you wanted a JJA target period, you would use 'Jun-Aug'.\n",
+    "    'target': 'Jun',\n",
+    "\n",
+    "    # 'predictor_extent':\n",
+    "    #   The geographic bounding box of the climate model data you want to download.\n",
+    "    #   This field is defined by a python dictionary with the keys \"north\", \"south\",\n",
+    "    #   \"east\", and \"west\", each of which maps to a python integer representing the \n",
+    "    #   edge of a bounding box. i.e., \"north\" will be the northernmost boundary,\n",
+    "    #   \"south\" the southernmost boundary.\n",
+    "    #   Example: {\"north\": 90, \"south\": -90, \"east\": 0, \"west\": 180}\n",
+    "    'predictor_extent': {\n",
+    "        'west': -80, \n",
+    "        'east':  -62,\n",
+    "        'south': -60, \n",
+    "        'north': -12,\n",
+    "      },\n",
+    "\n",
+    "    # 'predictand_extent':\n",
+    "    #   The geographic bounding box of the observation data you want to download.\n",
+    "    #   This field is defined by a python dictionary with the keys \"north\", \"south\",\n",
+    "    #   \"east\", and \"west\", each of which maps to a python integer representing the \n",
+    "    #   edge of a bounding box. i.e., \"north\" will be the northernmost boundary,\n",
+    "    #   \"south\" the southernmost boundary.\n",
+    "    #   Example: {\"north\": 90, \"south\": -90, \"east\": 0, \"west\": 180}\n",
+    "    'predictand_extent': {\n",
+    "        'west':  -80,\n",
+    "        'east': -62,\n",
+    "        'south': -60,\n",
+    "        'north': -12,\n",
+    "      },\n",
+    "\n",
+    "    'use_dlauth': True,\n",
+    "\n",
+    "    # 'filetype':\n",
+    "    #   The filetype to be downloaded. for now, it saves a lot of headache just to set this equal\n",
+    "    #   to 'cptv10.tsv' which is a boutique plain-text CPT filetype based on .tsv + metadata.\n",
+    "    'filetype': 'cptv10.tsv'\n",
+    "}\n",
+    "\n",
+    "cpt_args = { \n",
+    "    'transform_predictand': None,  # transformation to apply to the predictand dataset - None, 'Empirical', 'Gamma'\n",
+    "    'tailoring': None,  # tailoring None, 'Anomaly'\n",
+    "    'cca_modes': (1,3), # minimum and maximum of allowed CCA modes \n",
+    "    'x_eof_modes': (1,8), # minimum and maximum of allowed X Principal Componenets \n",
+    "    'y_eof_modes': (1,6), # minimum and maximum of allowed Y Principal Components \n",
+    "    'validation': 'crossvalidation', # the type of validation to use; only 'crossvalidation' is supported for now\n",
+    "    'drymask': False, #whether or not to use a drymask of -999\n",
```

Comment:
It looks like you copied the `cpt_args` block from `pycpt-station.ipynb`, which I see is out of date.  The `drymask` arg was never released, and has been replaced in the upcoming release by `drymask_threshold`. Also `skillmask_threshold` is missing, and `scree` is still there but should have been removed. Please use the `cpt_args` block from `pycpt-operational.ipynb` instead, and I will do the same in `pycpt-station.ipynb`.